assemble/pom.xml

```diff
@@ -171,6 +171,9 @@
             <excludes>
               <exclude>bin-LICENSE</exclude>
               <exclude>bin-NOTICE</exclude>
+              <exclude>nbproject/**</exclude>
+              <exclude>nb-configuration.xml</exclude>
+              <exclude>nbactions.xml</exclude>
```

Comment:
Ok, the spirit of my question was just "netbeans is creating these itself", but thanks for the explanation!